scrunch/expressions.py

```diff
@@ -449,8 +449,9 @@ def adapt_multiple_response(ds, var_url, values):
     expression.
     """
     # convert value --> column and change ids to aliases
-    subvariable_urls = ds.variables.index[var_url]['subvariables']
-    variable = ds.variables.index[var_url].entity
+    var_index = ds.variables.index
+    subvariable_urls = var_index[var_url]['subvariables']
+    variable = var_index[var_url].entity
     aliases = []
     for sub_var in subvariable_urls:
         subvar_id = variable.subvariables.index[sub_var]['id']
```

Comment:
Note that here you're hitting the same `variable.subvariables` catalog each time inside the above iteration for each subvariable.
You can change this block like this:

```
for subvar_alias, subvar_tup in variable.subvariables.by('alias'):
  var_id = subvar_tup['id']
```

And you have all subvariables IDs and aliases in only one request. Even better, is it possible to know the same information fromt https://github.com/Crunch-io/scrunch/blob/1c92a46cbd713797c24f9cda85def7c78afea6e3/scrunch/expressions.py#L475 since it already has the subvariables information for the array variables https://github.com/Crunch-io/scrunch/blob/1c92a46cbd713797c24f9cda85def7c78afea6e3/scrunch/expressions.py#L430 this would avoid doing multiple requests per each variable in the expression.